README.md

```diff
@@ -8,6 +8,7 @@
     - [Setup](#setup)
     - [Prerequisites](#prerequisites)
     - [Usage](#usage)
```

Comment:
- [ ] The new expense page should have a back arrow (<) that takes the user to the related transactions list page.

  Your Implementation | Requirement
  :---:|:---:
  ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/c996173e-e86d-4227-a657-eada110c594c) | ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/728e565a-11c0-4bb1-8b86-349b62bcc935)